cli/popper/commands/cmd_search.py

```diff
@@ -258,9 +259,24 @@ def search_pipeline(repo_url, keywords, org_name, empty_query,
         else:
             if l_distance(keywords.lower(),
                           pipeline['name'].lower()) < 1:
+
                 temp = "{}/{}/{}" \
                     .format(org_name, repo_name, pipeline['name'])
 
+                readme_url = "https://raw.githubusercontent.com"
+                readme_url += "/{}/{}/master".format(org_name, repo_name)
+                readme_url += "/pipelines/{}/README.md".format(
+                    pipeline['name'])
+
+                r = requests.get(readme_url, headers=headers)
+                if r.status_code != 200:
+                    pass
+                else:
+                    content = str(BytesIO(r.content).getvalue(), 'utf-8')
+                    # print(type(content)
+                    content = "\n".join(content.split("\n")[:3])
+                    temp += "\n" + content + "..."
```

Comment:
is the goal here to display the content of a README in the `popper search` output? If that's the case, it might be better to move this to be part of the output of `popper info`. Issue #145 was created a while a got to track this new feature

Reply:
Yes the goal was to add 2-3 lines of a pipeline's README file in the popper search output. 
I will move this functionality to popper info. :+1:

Reply:
in order to close #145, the only remaining thing to do would be to add the `--readme` flag, to indicate that users want to search for keywords in the README too, not just on the name of the pipeline.

Reply:
And for when to display this:

  * `search`. show the match on the README when the `--readme` flag is given. We can initially show the entire README content. We can then open an issue where just the matching lines on the README are shown, to mimic the behavior of `grep`.
  * `info`. always show the content on the README

Reply:
> We can initially show the entire README content
What if we only show 2-3 lines from the README (when the --readme flag is given) and ask them to use popper info if the want to see the complete content?

Reply:
sounds good!